### PR TITLE
Added admin page.

### PR DESCRIPTION
**Pull request by Corvax357**
Used cookies to check Master user, because log-adm-page redirects to adm-page "refreshing" all global variables. Is it ok?)

### CODE REVIEW

**Comment by ostaptan** [comment]
Definitely!

File: javascripts/admin-page.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Admin-page users
+var Master = {login :	"Master", pwd : "Master357"};
+var Admin = {login : "Admin", pwd : "Admin357"};
```
Comment (ostaptan) [inline javascripts/admin-page.js]: use 2 spaces tab

File: javascripts/admin-page.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+window.onload = function checkUser() {
+    var cookies = document.cookie;
+    if (cookies !== "") {
+        MasterMode = true;
+    } 
+}
+
+// Showing Users list if signed in as Master (MasterMode=true)      
+function showUsers() {
+    if(MasterMode) {
```
Comment (ostaptan) [inline javascripts/admin-page.js]: USE SPACES! LAST WARNING! maybe copy-pasted????

Comment (Corvax357) [inline javascripts/admin-page.js]: Definitely not, ll be more patient(

**Pull request merged**